config/config.yml.example

```diff
@@ -59,7 +59,7 @@ movement:
 plugins:
     # Do not automatically load the specified plugins
     exclude:
-        - socket
+        - pluginname
```

Comment:
I think a lot of people are just going to rename this file, so i would leave exclude an an empty array. maybe update the comments to show it should be the plugin name.